{"name": "test_company_overview_without_symbol_parameter", "status": "broken", "statusDetails": {"message": "TypeError: assert_equal() got an unexpected keyword argument 'msg'", "trace": "self = <test_company_overview.Test_CompanyOverview testMethod=test_company_overview_without_symbol_parameter>\n\n    def test_company_overview_without_symbol_parameter(self):\n        self.logger.info(\"test_time_series_daily_invalid_symbol started\")\n        url = self.base_url+\"function=OVERVIEW&symbol=&apikey=\"+self.apiKey\n        self.logger.info(\"url : \" + url)\n        response = requests.get(url)\n        response_json = response.json()\n        self.logger.info(\"response : \" + response.text)\n        # response assertion\n>       self.assertion.assert_equal(response_json['Error Message'],\n                         \"Invalid API call. Please retry or visit the documentation (https://www.alphavantage.co/documentation/) for OVERVIEW.\",\n                         msg=\"response not meeted expected error\" + response_json['Error Message'])\nE       TypeError: assert_equal() got an unexpected keyword argument 'msg'\n\nTests/test_company_overview.py:83: TypeError"}, "attachments": [{"name": "log", "source": "c00e5894-ede7-4918-91ef-26b716b976ec-attachment.txt", "type": "text/plain"}], "start": 1611645618115, "stop": 1611645619209, "uuid": "89c718a5-2aca-4735-bf4c-78eb97e00a64", "historyId": "77c348c1a49e652963e043d84dbebfb1", "testCaseId": "fe18ad514249c36f6a8a46de5a6065c7", "fullName": "Tests.test_company_overview.Test_CompanyOverview#test_company_overview_without_symbol_parameter", "labels": [{"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_company_overview"}, {"name": "subSuite", "value": "Test_CompanyOverview"}, {"name": "host", "value": "san-ThinkPad-T480"}, {"name": "thread", "value": "268176-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_company_overview"}]}